src/com/example/ui.cljs

```diff
@@ -30,24 +33,63 @@
 
 (def ui-header (comp/factory Header))
 
+(defn field [{:keys [label valid? error-message input-class id] :as props}]
+  (let [input-props (-> props
+                        (dissoc :label :valid? :error-message :input-class :options))]
+    (div
+     (dom/label {:htmlFor label} label)
+     (input-class input-props)
+     (dom/div {:classes [(when valid? "hidden")]}
+              error-message))))
+
+(defsc Question [this {:question/keys [id label options]}]
+  {:query [:question/id
+           :question/label
+           :ui/selected
```

Comment:
`:ui/` namespaced keys are used only on the fulcro side. they will never be sent to pathom